src/D2L.Bmx/OktaAuthenticator.cs

```diff
@@ -55,67 +55,27 @@ bool bypassBrowserSecurity
 		}
 
 		var orgUrl = GetOrgBaseAddress( org );
-		var oktaAnonymous = oktaClientFactory.CreateAnonymousClient( orgUrl );
```

Comment:
a side benefit of wrapping things in dedicated private method is that it exposes and prevents when we declare/define variables too early, like here